demos/tgen/test_tgen_demo.py

```diff
@@ -3,26 +3,12 @@
 import pydpu
 import pytest
 import snappi
-
-
-import os
-
-# for name, value in os.environ.items():
-#     if name == 'TGEN1_PASS':
-#         print("{0}: {1}".format(name, '***********'))
-#     else:
-#         print("{0}: {1}".format(name, value))
-
-
-ROOT_PASSWORD = os.getenv('ROOT_PASSWORD')
-
-
-
+import requests
 
 from testbed import *
 
-
-
+HOME_FOLDER = r'/home/opi/actions-runner/_work/opi-poc/opi-poc'
```

Comment:
I don't like globals

Reply:
i need to play with the action runner to see what is the relative path, and make it work when ran via git clone manually and via action runner, it's on TODO list